.editorconfig

```diff
@@ -0,0 +1,12 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+indent_style = space
+indent_size = 2
```

Comment:
are you sure it completely follows our code style? I think we have indent_size set to 4 currently, right?

Reply:
I'll re-check :)